### ISSUE TITLE
Victrom BMV

### ISSUE THREAD

**Issue by Zippa1204** [njfaria/SolPipLog#65]
Hi,

I have a Victron BMV702. Does it also work?

Am I missing something how to connect, as I do not see the USB port in the selection?

Gert

**Comment by Ricknanon** [comment]
Hello. 
You have to install Victron battery monitor (from same repo) and this app you will find the usb to connect. You will need as well to enable mqtt if you want the threshold to work with solpiplog ( ie at 70% of discharge go back to grid in solpiplog)
Have a good day.

**Comment by Zippa1204** [comment]
Hi,

Yes, I have the APP, but when I want to connect, the USB port listed seems odd - or is it correct

Gert

**Comment by Ricknanon** [comment]
Yes it is those odd names. Try to connect any the app will tell you if it is ok.

**Comment by Zippa1204** [comment]
Hi, OK I opted for Daly BMS.
However, see the cell voltages.....this is creating bad graphs on Emoncms.
![DalyBMS](https://user-images.githubusercontent.com/88541798/129595916-2ad20ab0-e6d5-4d33-9a2e-819b562adefe.JPG)

**Comment by Ricknanon** [comment]
It is not supposed to work with Daly if you have a BMV !

**Comment by njfaria** [comment]
@Zippa1204 if you got a BMV you should use https://github.com/njfaria/bmvlog

**Comment by Zippa1204** [comment]
Hi,

Yes, aware of that. Justto clarify:
My Friend has Victron BMV.
I have Daly BMS.

I have shared the pic from my Daly BMS, using the DalyLog App.

I also noted that the SOC switching is not happening when I selected it. I set to 20%, but batteries went down to 15%, when inverter switched back to grid.

Gert

**Comment by Ricknanon** [comment]
Ok 👍 
I ll try on my side for soc. 
Did you check on solpiplog tha soc is ok when running bmvlog. 
They need to run both and to be mppt connected. 
Rick

Envoyé de mon iPhone

**Comment by Zippa1204** [comment]
Hi,

Yes, both runnig on same Raspberry, as well as MQTT running on both apps. On SolPipLog, I can see the SOC value passed from DalyLog - so that seems to work correct.

Gert

**Comment by Ricknanon** [comment]
I will change the value on mine to check. 

Envoyé de mon iPhone

**Comment by Ricknanon** [comment]
Hello i tried on my side and it is working fine.  
« Soc is low «  is depicted on solpiplog switch tab. 
And my inverter switched to utility.

**Comment by Zippa1204** [comment]
Hi,

OK, Maybe I am doing something wrong then, as mine never shows SOC is low.

I saw that you do not have access to Daly BMS on Pi etc - I am willing to give you VNC access to my Pi should you wish?

Gert

**Comment by njfaria** [comment]
just invite solpiplog at gmail dot com to your vnc

Zippa1204 ***@***.***> escreveu no dia quinta, 19/08/2021
à(s) 17:43:
